### ISSUE TITLE
Kotlin support

### ISSUE THREAD

**Issue by bbetter** [android10/frodo2#7]
Frodo has issue with kotlin support is it somehow resolved in this version?

**Comment by m-ivanov** [comment]
kotlin methods annotated with  @RxLogFlowable do not works

```
    @RxLogFlowable
    Flowable<Integer> numbers() {
        return Flowable.just(1, 2, 3, 4);
    }

```

**Comment by android10** [comment]
Yes, it is true, the issue is that I'm trying to get some help to trigger the aspect compile with Kotlin....

Any help/contribution very appreciated it. 

Check this issue: https://github.com/android10/frodo2/issues/1
